d2/src/main/java/com/linkedin/d2/balancer/strategies/relative/TrackerClientState.java

```diff
@@ -0,0 +1,116 @@
+/*
+   Copyright (c) 2020 LinkedIn Corp.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package com.linkedin.d2.balancer.strategies.relative;
+
+import com.linkedin.util.degrader.ErrorType;
+import java.util.Map;
+
+
+/**
+ * Keeps the state of each tracker client for a partition
+ */
+public class TrackerClientState {
```

Comment:
maybe ServerHealthState instead?

Reply:
Hmm the concept of "server" is not explicitly mentioned, if the state is just tracking the state of each TrackerClient, it has a mapping relationship, I think TrackerClientState should be good